test/test_one_dimensional_shallow_water_model.jl

```diff
@@ -0,0 +1,49 @@
+# # One dimensional shallow water example
+
+using Oceananigans, Oceananigans.Advection
+
+using Oceananigans.Models # should import the shallow water model stuff
+#include("../src/Models/ShallowWaterModels/ShallowWaterModels.jl")
+using .ShallowWaterModels: ShallowWaterModel
+
+grid = RegularCartesianGrid(size=(64, 1, 1), extent=(2π, 2π, 2π))
+
+model = ShallowWaterModel(        grid = grid,
+                          architecture = CPU(),
+                             advection = nothing, 
+                              coriolis = nothing, 
+                             solution = nothing,
+                              tracers = nothing#,
```

Comment:
Trailing commas are ok, so we don't need the comment here

Reply:
Comment is gone